.github/workflows/build-deploy-pudl.yml

```diff
@@ -9,7 +9,7 @@ on:
 
 env:
   GCP_BILLING_PROJECT: ${{ secrets.GCP_BILLING_PROJECT }}
-  GITHUB_REF: ${{ github.ref_name }} # This is changed to dev if running on a schedule
+  BUILD_REF: ${{ github.ref_name }} # This is changed to dev if running on a schedule
```

Comment:
It seemed like our use of `GITHUB_REF` as a variable name was shadowing an existing environment variable. When we re-set `GITHUB_REF` to equal `dev` below, it would be `dev`, but when it wasn't overridden, `GITHUB_REF` would contain the full ref e.g. `/refs/heads/main` rather than the shortened version (`main`) that's supposed to be found in `github.ref_name`.  In any case, re-using the name and making it refer to something with different properties was confusing, so I renamed it to `BUILD_REF`